.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,37 @@
+name: deploy
+
+on:
+  push:
+    branches: [main]
+    tags: [v*]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
So @nlordell  - I don't need to do anything here, this secret will be available and if I tag the branch an image will be pushed to docker hub?

Reply:
From the [docs](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret):

> At the start of each workflow run, GitHub automatically creates a unique `GITHUB_TOKEN` secret to use in your workflow. You can use the `GITHUB_TOKEN` to authenticate in a workflow run.